### PR TITLE
update osg-qt from Qt4 tag to master branch(Qt5)

### PR DESCRIPTION
**Pull request by ZgblKylin**
**Describe the pull request**

- What does this PR do?

    - Remove unused dependency of protobuf.

    - Update repo from tag `Qt4` to latest master commit (Qt5 version).

- Which triplets are supported/not supported? Have you updated the CI baseline?

    - Support all triplets for previous version.

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?

    - Yes

### CODE REVIEW

**Comment by JackBoosY** [comment]
cc @Neumann-A for review this PR.

**Comment by ZgblKylin** [comment]
@Neumann-A pipeline job x64_windows_static was [abandoned with the output](https://dev.azure.com/vcpkg/public/_build/results?buildId=45801&view=logs&j=64fd9cb1-d359-5304-34d5-f67d386745f9&t=36551c36-57b4-575f-f053-0712b8ed98c0&l=1):

> This job was abandoned. We have detected that logs from the agent may have not finished uploading. We have included our in-memory record of all log lines uploaded before we lost contact with the agent:

How to restart the job?

**Comment by JackBoosY** [comment]
@ZgblKylin Sorry for late, can you please merge your branch with master?

**Comment by ZgblKylin** [comment]
@JackBoosY Sorry for late, I've rebased this branch onto master, and fixed baseline version (from Qt4#3 to Qt5#4)

Moreover, I've tried migrate to manifest file, which failes in pipeline [Create Diff](https://dev.azure.com/vcpkg/public/_build/results?buildId=50280&view=logs&j=67167230-dc6a-522f-6bed-e189fa130338&t=b4e18006-8dc4-525c-52dc-d5fd5bc99669)

File: ports/osg-qt/portfile.cmake [hunk 1]
```diff
@@ -1,12 +1,13 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO openscenegraph/osgQt
-    REF Qt4
-    SHA512 426a4ba88f680978d24817248b99c68cafa4517144e6e3d2480612870c4a224bb955539cacb438274d4ee1c93c36d94f8437d142070b2ecde2b81517bf357e71
+    REF 2cb70673a4e83a618290e7ee66d52402a94ec3f6
```
Comment (JackBoosY) [inline ports/osg-qt/portfile.cmake:4]: Is this a develop version?

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: ports/osg-qt/portfile.cmake (hunk 1 above)
Comment (ZgblKylin) [reply ports/osg-qt/portfile.cmake:4]: osgQt is a simple wrapper of OpenSceneGraphic library, it does not have any new tag since Qt5 version

**Review by JackBoosY — APPROVED** [review]

File: ports/osg-qt/CONTROL [hunk 2]
```diff
@@ -1,5 +1,5 @@
 Source: osg-qt
-Version: Qt4
-Port-Version: 3
-Description: osgQt - Qt project for making use of OpenSceneGraph(OSG) 
-Build-Depends: osg, protobuf, qt5-base[core]
\ No newline at end of file
+Version: Qt5
+Port-Version: 4
```
Comment (strega-nil) [inline ports/osg-qt/CONTROL:3]:
```suggestion
```

File: versions/baseline.json [hunk 3]
```diff
@@ -4449,8 +4449,8 @@
       "port-version": 6
     },
     "osg-qt": {
-      "baseline": "Qt4",
-      "port-version": 3
+      "baseline": "Qt5",
+      "port-version": 4
```
Comment (JackBoosY) [inline versions/baseline.json:4453]:
```suggestion
```

File: versions/o-/osg-qt.json [hunk 4]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "81a81a658f8396c5a7d37931b93ed45207ba3278",
+      "version-string": "Qt5",
+      "port-version": 4
```
Comment (JackBoosY) [inline versions/o-/osg-qt.json:6]:
```suggestion
      "port-version": 0
```

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: versions/baseline.json [hunk 5]
```diff
@@ -4449,8 +4449,7 @@
       "port-version": 6
     },
     "osg-qt": {
-      "baseline": "Qt4",
-      "port-version": 3
+      "baseline": "Qt5",
```
Comment (JackBoosY) [inline versions/baseline.json:4452]:
```suggestion
      "baseline": "Qt5",
      "port-version": 0
```

File: versions/o-/osg-qt.json [hunk 6]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "81a81a658f8396c5a7d37931b93ed45207ba3278",
```
Comment (JackBoosY) [inline versions/o-/osg-qt.json:4]:
```suggestion
      "git-tree": "912b87e36bf6db14319d83e11d91286b6921675f",
```

**Review by JackBoosY — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for your contribution!

**Pull request merged**